<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projet_button</class>
 <widget class="QDialog" name="projet_button">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>691</width>
     <height>491</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_license_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>441</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="Afficher_2">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_quit_projet">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>360</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Liste des licenses</string>
     </property>
     <widget class="QPushButton" name="pushButton_sup_projet">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>60</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_projet">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>401</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN a supprimer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CINsup_projet">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CINrec_projet">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rec_projet">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>150</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Entrer CIN a rechercher:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>-8</y>
       <width>691</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>pushButton_quit_projet</zorder>
    <zorder>groupBox_3</zorder>
   </widget>
   <widget class="QWidget" name="Ajouter_2">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_projet">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter une license</string>
     </property>
     <widget class="QLabel" name="label_projet_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ajouter_projet">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CIN_projet">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nom_projet">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Pr_projet">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du licenses:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nl_projet">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_8">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mail_projet">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>-8</y>
       <width>681</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>groupBox_projet</zorder>
   </widget>
   <widget class="QWidget" name="Modifier">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_projet_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>Modifier</string>
     </property>
     <widget class="QLabel" name="label_projet_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CIN_projet_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nom_projet_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Pr_projet_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_12">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du licenses:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nl_projet_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_projet_13">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mail_projet_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mod_projet_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-8</y>
        <width>691</width>
        <height>421</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>label_projet_9</zorder>
     <zorder>label_projet_10</zorder>
     <zorder>label_projet_11</zorder>
     <zorder>lineEdit_CIN_projet_2</zorder>
     <zorder>lineEdit_Nom_projet_2</zorder>
     <zorder>lineEdit_Pr_projet_2</zorder>
     <zorder>label_projet_12</zorder>
     <zorder>lineEdit_nl_projet_2</zorder>
     <zorder>label_projet_13</zorder>
     <zorder>lineEdit_mail_projet_2</zorder>
     <zorder>pushButton_mod_projet_2</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
